petstore/frontend/react/src/App.css

```diff
@@ -1,28 +1,36 @@
+.about {
```

Comment:
Almost all changes in this file are from alphabetizing the rules and running Prettier on the file to format it.